src/stores/RoomListStore.tsx

```diff
@@ -52,17 +56,35 @@ export class RoomListStore {
     return this._roomOverviews;
   }
 
+  get imgUrl() {
+    return this._imgUrl;
+  }
+
   get isSuccessCreate() {
     return this._isSuccessCreate;
   }
+
   get isSuccessGet() {
     return this._isSuccessGet;
   }
 
+  isSuccessImportImage() {
```

Comment:
getter로 하는게 좋아보이네요. 

언제 getter를 언제 쓰지 말아야하는지 간단히 설명하자면 아래와 같습니다.
> 수행시간이 오래걸리는 연산(ex: O(1) 연산이 아닌경우), 내부 상태를 변화하는 액션인경우

그런데 이 함수는 위에 해당되지 않아 getter를 적용하는게 좋아보입니다. 

```suggestion
  get isImageImported(): boolean {
```